### PR TITLE
add Gallery transitions benchmark to dashboard

### PR DESCRIPTION
**Pull request by yjbanov**
Bonus: make the UI more compact to fit more cards on the screen

Fixes https://github.com/flutter/flutter/issues/3896

screen shot 2016-05-13 at 2 14 20 pm

/cc @HansMuller @sethladd @devoncarew

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -168,6 +166,21 @@
...
 
+  <template id="__transition_perf_tmpl">
+    <div class="card from-firebase">
+      <div class="card-inside">
+        <div class="metric-type">
+          Gallery Transitions
+          <span class="metric-name-wrapper">(<span class="metric-name"></span>)</span>
+        </div>
+        <div class="frames" style="height: 120px; margin-top: 15px;">
+          <div class="frame-budget-indicator" style="height: 40px">100,000ms</div>
```
Comment (HansMuller) [inline web/index.html]: The thresholds you want to show here are 100ms (the data is in microseconds).

Comment (yjbanov) [inline web/index.html]: Yep. Will fix.

Comment (yjbanov) [inline web/index.html]: Fixed.

**Comment by HansMuller** [comment]
LGTM. I guess we've got some transition-times to work on.

**Pull request merged**